db/bindata.go

```diff
@@ -90,7 +90,7 @@ func sql001_create_to_seconds_functionSql() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "sql/001_create_to_seconds_function.sql", size: 230, mode: os.FileMode(420), modTime: time.Unix(1515665547, 0)}
+	info := bindataFileInfo{name: "sql/001_create_to_seconds_function.sql", size: 230, mode: os.FileMode(436), modTime: time.Unix(1519124886, 0)}
```

Comment:
Why is the file mode changing?

Reply:
I assume this was change was generated by go-bindata. So this isn't even a _real_ file. Very odd that you would want the fake file permission of group execute for a fake file! 

We have a story to remove the use of go-bindata since there are some trust issues with the repository's owner.

Reply:
Yeah it's probably part of an elaborate hack.